package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "propsd",
   "version": "0.0.1",
-  "description": "A local daemon for loading static and dynamic properties",
+  "author": "Rapid7 <coreservices@rapid7.com>",
+  "contributors": [
+    "David Greene <dgreene@rapid7.com>",
+    "John Manero <jmanero@rapid7.com>",
+    "Frank Mitchell <fmitchell@rapid7.com>",
+    "Richard Van Buren <rvanburen@rapid7.com>"
+  ],
+  "description": "A local properties management service",
   "main": "lib/index.js",
-  "directories": {
-    "doc": "docs",
-    "bin": "bin"
-  },
   "scripts": {
-    "test": "mocha",
-    "lint": "eslint lib/**"
+    "test": "./node_modules/.bin/mocha",
+    "lint": "./node_modules/.bin/eslint bin lib test"
```

Comment:
You don't need to specify `./node_modules/.bin` here. When you execute `npm run *` it executes with `node_modules/.bin` added to the $PATH.